test/gtest/cache.cpp

```diff
@@ -145,13 +147,19 @@ void check_cache_str()
     CHECK(p.filename().string() == name + ".o");
 }
 
-int main()
+//Google Test
+TEST(TestCache, BasicAssertions)
 {
+    auto marker = 0;
     check_cache_file();
     check_cache_str();
+    ++marker;
+    EXPECT_EQ(marker, 1);
 #if MIOPEN_ENABLE_SQLITE
-    check_bz2_compress();
-    check_bz2_decompress();
+    check_bz2_compress();    
+    check_bz2_decompress();    
     check_kern_db();
+    ++marker;
+    EXPECT_EQ(marker, 2);
```

Comment:
Out tests (including this cache.cpp) use our own macro `EXPECT` throughout the sources for validation:
(https://github.com/ROCmSoftwarePlatform/MIOpen/blob/f09132964d628ffb649f4a6ee14dadfc933a7a69/test/test.hpp#L76-L82)
If we want to port it to GoogleTest, then shall we also update all these checks with the ones provided by the GoogleTest framework?

Reply:
@atamazov , Seems EXPECT() is doing similar things, my understanding is yes, we may need to update the Macros with those provided by GoogleTest, does this make sense?